index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<meta charset="utf-8">
+<link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+<h1>todos</h1>
+
+<div class="table">
+	<div class="div_first"><label> ˅ </label><em> What needs to be done?</em></div>
+	<div class="div_second"><input type="checkbox" checked>to learn HTML</div>
+	<div class="div_3"><input type="radio">to learn CSS</div>
+	<div class="div_3"><input type="radio">to learn JS</div>
+
+	<div class="div_bottom">
+		<div class="div_bottom_1">2 items left</div>
+		<div class="div_bottom_bordered">All</div>
+		<div class="div_bottom_2">Active</div>
+		<div class="div_bottom_2">Completed</div>
```

Comment:
please change class names to have it understandable for others

Reply:
changed